test/e2e/performanceprofile/functests/utils/profilesupdate/profile_update.go

```diff
@@ -34,7 +35,7 @@ func UpdateIsolatedReservedCpus(isolatedSet performancev2.CPUSet, reservedSet pe
 
 	err = ApplyProfile(updatedProfile)
 	if err == nil {
-		testlog.Infof("successfully updated performance profile %q with new isolated cpus set: %q and new reserved cpus set: %q", profile.Name, updatedProfile.Spec.CPU.Isolated, updatedProfile.Spec.CPU.Reserved)
+		testlog.Infof("successfully updated performance profile %q with new isolated cpus set: %q and new reserved cpus set: %q", profile.Name, cpuset.MustParse(string(*updatedProfile.Spec.CPU.Isolated)), cpuset.MustParse(string(*updatedProfile.Spec.CPU.Reserved)))
```

Comment:
I'd just deference the pointers here. The value added by `MustParse` is not obvious.
```
testlog.Infof("successfully updated performance profile %q with new isolated cpus set: %q and new reserved cpus set: %q", profile.Name, string(*updatedProfile.Spec.CPU.Isolated), string(*updatedProfile.Spec.CPU.Reserved))
```